packages/components/src/styles/interactive/interactive.declaration.ts

```diff
@@ -0,0 +1,22 @@
+import type { Style } from '../../declaration';
+
+export default {
+  styleName: 'sd-interactive',
+  summary: 'Generates basic styles for interactive elements.',
+  status: 'experimental',
+  since: '1.9',
```

Comment:
Probably needs to be changed before merging, we are currently at 1.9. BTW: What is the next version? 2.0 or 1.10?

Reply:
@MarcMatthiae as it stays a feature, it will become 1.10. major version bumps (2.0) only apply for breaking changes.